notes/raindrops.md

```diff
@@ -1,15 +1,25 @@
 Congratulations on passing all the tests!
 
-If you cared to, for further exercise, you might try an [[iter](https://doc.rust-lang.org/std/primitive.slice.html#method.iter)], a [[map](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.map)] and a [[collect](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect)]. Using a list of [[tuples](https://doc.rust-lang.org/stable/rust-by-example/primitives/tuples.html)] to iterate over, e.g. `(3, "Pling")`, would work well with that approach. There are other ways that a `match` on a `tuple` could work, for example...
+If you cared to, for further exercise, you might try an 
+[iter](https://doc.rust-lang.org/std/primitive.slice.html#method.iter),
+a [map](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.map) and a
+[collect](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect). 
+Using a list of 
+[tuples](https://doc.rust-lang.org/stable/rust-by-example/primitives/tuples.html)
+to iterate over, e.g. `(3, "Pling")`, would work well with that approach. There 
+are other ways that a `match` on a `tuple` could work, for example...
 
 ```rust
+
 pub fn raindrops(n: u32) -> String {
     let output = match (n % 3 == 0, n % 5 == 0, n % 7 == 0) {
         (true, false, false) => "Pling",
 //and so on
+
 ```
```

Comment:
Ah no, I'm so sorry, when I meant "blank lines around code blocks", I meant like you already did in the previous commit but not always in your other notes: blanks lines before and after entire code blocks, *fences included*... So simply remove those two new blank lines you just added.

I wanted to give you a suggestion using GitHub's review system so you could commit it easily, but as there are fences in the code I'd like to change, it seems to conflict with the system itself :rage:.